Main.java

```diff
@@ -1,3 +1,5 @@
+import java.util.Scanner;
+
 /**
  *
  * @author 

```

Comment:
Don't forget to fill out the code header. It should include a description of the code and also your name.